python/CMakeLists.txt

```diff
@@ -114,3 +108,7 @@ if(NOT MPI4PY_COMMAND_RESULT)
 else()
   message(FATAL_ERROR "mpi4py could not be found.")
 endif()
+
+set_target_properties(cpp PROPERTIES INSTALL_RPATH_USE_LINK_PATH TRUE)
```

Comment:
This might break relocatable builds, e.g. wheels. See how it work in Basix.

Reply:
I/we to look at this more closely.

Basix are relocatable, I agree. Not convinced about DOLFINx.

Reply:
> I/we to look at this more closely.
> 
> Basix are relocatable, I agree. Not convinced about DOLFINx.

Could it affect conda?

Reply:
@minrk Do you have something to add here? I'm honestly not sure how this setting `INSTALL_RPATH_USE_LINK_PATH` affects downstream packaging.